templates/layout.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html>
   <head>
-    <title>{{page_title}}</title>
+    <title>{% if page_title %}{{page_title}}{% else %}Hedy{% endif %}</title>
```

Comment:
In the R description you write: "If a key in the dict doesn't exist we automatically return the title of the English homepage ("Hedy - A gradual program language")"

but this seems like it will be 'Hedy' when there is no title? (Not that it matters a lot to me either way :))

Reply:
We return the start title for "en" if for some reason we can't find the key or lang in the dict. But if for some reason we add no `page_title` at all to the `render_template` we will still show "Hedy" as title (and prevent the page from crashing).

Reply:
Ah got it!